recipe/make_pkg_config.py

```diff
@@ -0,0 +1,62 @@
+import os
+from pathlib import Path
+
+tpl = """\
+prefix={PREFIX}
+exec_prefix=${{prefix}}
+libdir=${{prefix}}/lib
+sharedlibdir=${{libdir}}
+includedir=${{prefix}}/include
+
+Name: {NAME}
+Description: {DESCRIPTION}
+Version: {VERSION}
+Cflags: -I${{includedir}}
+Libs: -L${{libdir}} -l{NAME}
+"""
+
+prefix = Path(os.environ["PREFIX"])
```

Comment:
when on windows, "/Library" needs to be append to prefix: There should also be a `LIBRARY_PREFIX` environment variable in conda-build recipes on windows that would point to this correct path

```
prefix = Path(os.environ.get("LIBRARY_PATH", os.environ["PREFIX"]))
```

Reply:
I didn't realize pkg-config was a thing on Windows! This doesn't generate files for Windows at all.

If you know what they should be, a PR updating the script and adding it to bld.bat would be much appreciated.

Reply:
I opened https://github.com/conda-forge/mumps-feedstock/issues/129 to track this, as I experienced this today.